docs/core-fabric.md

```diff
@@ -457,9 +478,8 @@ Simple:
 
 ```yaml
 persistence:
-  type: sql
+  type: sqlite
```

Comment:
is this accurate? should `persistence` be something else?

Reply:
In general it should refer to the name of the persistence config. But here it explains more what the format of each config is, that's why I left it like that.